protos/bento/protos/services.proto

```diff
@@ -63,22 +63,29 @@ message DropSimulationReq {
 }
 message DropSimulationResp {}
 
-message StepSimulationReq {}
+message StepSimulationReq {
+  // Name of the simulation to step
+  string name = 1;
+}
 message StepSimulationResp {}
 
 message GetAttributeReq {
+  // Name of the simulation to retrieve the attribute from
+  string name = 1;
```

Comment:
Can this be renamed to `sim_name` to disambiguate from attribute name?

Reply:
Renamed in 407e3f8